sbd.spec

```diff
@@ -77,6 +82,9 @@ install -D -m 0755 src/sbd.sh $RPM_BUILD_ROOT/usr/share/sbd/sbd.sh
 %if %{defined _unitdir}
 install -D -m 0644 src/sbd.service $RPM_BUILD_ROOT/%{_unitdir}/sbd.service
 install -D -m 0644 src/sbd_remote.service $RPM_BUILD_ROOT/%{_unitdir}/sbd_remote.service
+%else
+install -D -m 0755 src/sbd_helper $RPM_BUILD_ROOT/%{_initrddir}/sbd_helper
+install -D -m 0755 src/sbd_remote_helper $RPM_BUILD_ROOT/%{_initrddir}/sbd_remote_helper
```

Comment:
I suspect that for systemd, different units were added so the Before/PartOf/RequiredBy could be different. But I'm not sure that was necessary.

Before is optional, so we could definitely list pacemaker and pacemaker_remote there. PartOf and RequiredBy should only matter if the other service is acted on, so I think it should be safe to list both there, too (since we prevent people from starting the sbd unit directly).